src/CSF.SRDashboard.Client/DTO/Azure/AzureMemberInfo.cs

```diff
@@ -30,7 +31,7 @@ public string Names
                 }
                 else
                 {
-                    return ((surname != null) ? surname + "," + givenName : null);
+                    return ((Surname != null) ? Surname + "," + GivenName : null);
```

Comment:
We probably want to return an empty string and not null.
```suggestion
                    return ((this.Surname != null) ? Surname + ", " + this.GivenName : string.Empty);
```